libraries/Microsoft.Bot.Builder.Dialogs/Prompts/OAuthPrompt.cs

```diff
@@ -259,7 +269,7 @@ private async Task SendOAuthCardAsync(ITurnContext turnContext, IMessageActivity
                                 {
                                     Title = _settings.Title,
                                     Value = link,
-                                    Type = turnContext.Activity.ChannelId == "msteams" ? ActionTypes.OpenUrl : ActionTypes.Signin,
+                                    Type = ActionTypes.Signin,

```

Comment:
There is another PR I just approved that does this a better way (still using OpenUrl for other channels is the right thing to do, I had gotten it backwards for teams)...so revert this part of your change, the other part looks good.

Reply:
thank you @Jeffders . Using OpenUrl was requested for Microsoft Teams only, it has been done in this PR https://github.com/Microsoft/botbuilder-dotnet/commit/eabb1181b7ae8e50ca0303cf4c852b8cf6843e69#diff-cc0d075891bd4a6dfa2e3b43f11cb016 . Do you think we should invert logic and use SignIn only in case of msteams context?

Reply:
@ky7m, What you would like to do with this PR? Should I close it?

Reply:
@cleemullins, if you are ok with this change https://github.com/Microsoft/botbuilder-dotnet/pull/1579/files/96e27f63f2d28da5edbbd18ff246262da11b2198 I can revert mine and submit code connected to fallback on mobile devices only.

Reply:
looks like, I was right :) https://github.com/microsoft/botbuilder-dotnet/commit/250a7a4767190b0f388b6704d165311519c6e101#diff-cc0d075891bd4a6dfa2e3b43f11cb016